Tests/AWSSDKSwiftCoreTests/AWSClientTests.swift

```diff
@@ -406,6 +406,35 @@ class AWSClientTests: XCTestCase {
             XCTFail(error.localizedDescription)
         }
     }
+    
+    func testSignedClient() {
+        let input = E()
+        let client = AWSClient(
```

Comment:
I recently added a helper function `createAWSClient()` which fills out all the parameters with default values. If you could use that instead that'd be great. Unless you want a specfic parameter set you can call `createAWSClient()` with nothing set. Means we have less client creation code filling up the tests.